standby/src/lib.rs

```diff
@@ -955,12 +931,11 @@ impl Standby {
     }
 
     /// Iterate over bystanders and remove the ones that match the predicate.
-    #[cfg_attr(feature = "tracing", tracing::instrument(level = "trace"))]
+    #[tracing::instrument(level = "trace")]
```

Comment:
Only `trace` level events are recorded inside, so there's no need for the instrument to be at the trace level.
```suggestion
    #[tracing::instrument]